app/main/util/render_utils.py

```diff
@@ -56,37 +54,21 @@ def get_download_filename(file):
     return None
 
 
-def manage_static_file(file, file_extension):
-    """Manage the file in the static directory."""
+def create_presigned_url(file):
     s3 = boto3.client("s3")
     bucket = current_app.config["RECORD_BUCKET_NAME"]
     key = f"{file.consignment.ConsignmentReference}/{file.FileId}"
-    files_directory = os.path.join(current_app.static_folder, "files")
-    static_file_path = ""
-
-    if file_extension == "pdf":
-        static_file_path = os.path.join(files_directory, "temp_file.pdf")
-    elif file_extension in ["png", "jpg", "jpeg"]:
-        static_file_path = os.path.join(
-            files_directory, f"temp_file.{file_extension}"
-        )
-    else:
-        static_file_path = os.path.join(
-            files_directory, f"temp_file.{file_extension}"
-        )
-
-    if os.path.exists(files_directory):
-        shutil.rmtree(files_directory)
 
-    os.makedirs(files_directory)
-
-    s3_file_object = s3.get_object(Bucket=bucket, Key=key)
-    file_content = s3_file_object["Body"].read()
+    try:
+        presigned_url = s3.generate_presigned_url(
+            "get_object", Params={"Bucket": bucket, "Key": key}, ExpiresIn=10
+        )
 
-    with open(static_file_path, "wb") as static_file:
-        static_file.write(file_content)
+    except Exception as e:
+        current_app.logger.error(f"S3 error: {e}")
```

Comment:
oh I see we abort now rather than letting it continue, ignore my message before then

Reply:
I think changing this error message to the one you recommended would be useful